### DISCUSSION TITLE
Effects related to Enhanced Domains ?

### DISCUSSION THREAD

**Discussion by Kerstin-Eigenherd** [jongpie/NebulaLogger#462] [Q&A]
Dear @jongpie ,

we are using Nebula Logger in our org and would like to know if we are still save when switching on Enhanced Domain Features?
If you made any changes in the past ensuring no relevants effects, could you let us know which version of Nebula Logger we need to have in place?

Your feedback is appreciated!
Kind regards, Kerstin

**Comment by jongpie** [comment]
Hi @Kerstin-Eigenherd - as far as I'm aware, there is no impact to Nebula Logger when enabling Enhanced Domains. I currently use Nebula Logger in scratch orgs, sandboxes, and prod orgs that all have Enhanced Domains enabled, and I have not seen any issues with it. If you happen to run into any issues, let me know, but I expect that everything will work without any major issues.
